app/apps/permits/api_queries_decos_join.py

```diff
@@ -354,15 +210,31 @@ def get_documents_with_folder_id(self, folder_id):
     def _get_decos_folder(self, decos_object):
         if not settings.USE_DECOS_MOCK_DATA:
             try:
-                decos_object_id = decos_object["content"][0]["key"]
+                #  Get all Decos object id's
+                list_of_decos_object_ids = [
+                    obj["key"] for obj in decos_object["content"]
+                ]
             except (KeyError, IndexError):
-                decos_object_id = False
+                list_of_decos_object_ids = False
                 response_decos_folder = False
 
-            if decos_object_id:
-                response_decos_folder = self.get_folders_with_object_id(decos_object_id)
+            if list_of_decos_object_ids and len(list_of_decos_object_ids) > 0:
+                response_decos_folder = {}
+                #  Get all folders for every id and merge them.
+                for decos_object_id in list_of_decos_object_ids:
+                    folder_with_object_id = self.get_folders_with_object_id(
+                        decos_object_id
+                    )
+                    response_decos_folder["count"] = (
+                        response_decos_folder.get("count", 0)
+                        + folder_with_object_id["count"]
+                    )
+                    response_decos_folder["content"] = [
+                        *response_decos_folder.get("content", []),
+                        *folder_with_object_id["content"],
```

Comment:
Bij de ene doe je wel een get met een default waarde, bij de andere niet. Waarom is daar een verschil?

Reply:
Changed!